packages/kotti-datepicker/src/DatePicker.vue

```diff
@@ -109,9 +109,11 @@ export default {
 	methods: {
 		monthPrevious() {
 			this.date = this.date.subtract(1, 'month')
+			this.$emit('KtDateSelected', this.date.toDate())
```

Comment:
Not a fan of the event name, but I’m assuming you chose this one for compatibility reasons

Reply:
ye exactly - don't  see anyone relying on v-model anyways and yoda has a wrapper around this that emits input allowing for the vue magical v-model